src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/SignatureVerification/Model/DCCSigningCertificate.swift

```diff
@@ -3,8 +3,11 @@
 //
 
 import Foundation
+import Security
+import CryptoKit
```

Comment:
Does that work with iOS 12.5? 🤔

Reply:
Yes, it is compiling, because the import is not used 🙈 removed it.